index.js

```diff
@@ -198,9 +198,25 @@ var publicAgent = {
   private_: function() { return agent; }
 };
 
-module.exports = global._google_trace_agent = publicAgent;
+// TODO: Update the documentation for the supplied options
+/**
+ * @param {object} options - These are currently ignored
+ */
+function Trace(options) {
+  if (!(this instanceof Trace)) {
+    return new Trace(options);
+  }
+}
+
+Trace.prototype.startAgent = function(projectConfig) {
```

Comment:
@ofrobots Is the preferred mechanism to retrieve an already initialized agent to call `startAgent` again? I believe we originally introduced the `get` function above to avoid this. Could the same be done here?